### ISSUE TITLE
[CLOSED] Bi-Weekly Discussion 2/20 - 3/6

### ISSUE THREAD

**Issue by bwwagner** [ephemerant/445-Final#36]
<a href="https://github.com/bwwagner"><img src="https://avatars4.githubusercontent.com/u/13163930?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [bwwagner](https://github.com/bwwagner)**
_Monday Feb 20, 2017 at 12:59 GMT_
_Originally opened as https://github.com/ephemerant/445/issues/36_

----

Continued from #34 

Changing to a bi-weekly period to match class meetings.

TODO:
 * Ionic calendar implementation
 * Add/Edit Menu Item modal Update
 * GPS update with Pusher
 * Truck List for Calendar View
 * Stop Broadcast with duration to broadcast
 * Meeting on Tuesday
 * Abstract for IUS Student Conference

**Comment by bwwagner** [comment]
<a href="https://github.com/bwwagner"><img src="https://avatars4.githubusercontent.com/u/13163930?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [bwwagner](https://github.com/bwwagner)**
_Monday Feb 20, 2017 at 15:52 GMT_

----

The abstract has to be 100 words or less, and match one of the formats listed at [Student Conference Website under Formats.](https://www.ius.edu/student-conference/formats.php) I've started a draft of this, but feel free to edit it.

We will probably have to do either a poster or tri-fold display with a discussion. Another option is a 15 minute presentation.

**Comment by bwwagner** [comment]
<a href="https://github.com/ephemerant"><img src="https://avatars7.githubusercontent.com/u/9091188?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [ephemerant](https://github.com/ephemerant)**
_Tuesday Feb 21, 2017 at 14:46 GMT_

----

Is everyone still able to do tonight at 7:30? I won't be able to stay for long, but I'll be there.

**Comment by bwwagner** [comment]
<a href="https://github.com/taayers"><img src="https://avatars5.githubusercontent.com/u/15347220?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [taayers](https://github.com/taayers)**
_Tuesday Feb 21, 2017 at 15:03 GMT_

----

Yeah, I'll be there.

**Comment by bwwagner** [comment]
<a href="https://github.com/bwwagner"><img src="https://avatars4.githubusercontent.com/u/13163930?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [bwwagner](https://github.com/bwwagner)**
_Tuesday Feb 21, 2017 at 17:15 GMT_

----

I should be able to make it to the meeting.

**Comment by bwwagner** [comment]
<a href="https://github.com/jrlytle"><img src="https://avatars5.githubusercontent.com/u/14928183?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [jrlytle](https://github.com/jrlytle)**
_Tuesday Feb 21, 2017 at 19:26 GMT_

----

Yep, I should be able to be there.

**Comment by bwwagner** [comment]
<a href="https://github.com/ephemerant"><img src="https://avatars7.githubusercontent.com/u/9091188?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [ephemerant](https://github.com/ephemerant)**
_Wednesday Mar 01, 2017 at 17:03 GMT_

----

I can't do a meeting tonight, but I should be good for the rest of the week if we need to schedule anything.

**Comment by bwwagner** [comment]
<a href="https://github.com/jrlytle"><img src="https://avatars5.githubusercontent.com/u/14928183?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [jrlytle](https://github.com/jrlytle)**
_Wednesday Mar 01, 2017 at 20:58 GMT_

----

The only other nights I can do this week are on the weekend. I have plans for Thursday and Friday.

**Comment by bwwagner** [comment]
<a href="https://github.com/ephemerant"><img src="https://avatars7.githubusercontent.com/u/9091188?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [ephemerant](https://github.com/ephemerant)**
_Wednesday Mar 01, 2017 at 20:59 GMT_

----

Turns out I can actually make it tonight. Sorry for the short notice, but would anyone be able to make it? If not, I understand, and I'm still good for the rest of the week.

**Comment by bwwagner** [comment]
<a href="https://github.com/bwwagner"><img src="https://avatars4.githubusercontent.com/u/13163930?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [bwwagner](https://github.com/bwwagner)**
_Sunday Mar 05, 2017 at 04:35 GMT_

----

OK, I have an embarrassing confession to make. The ionic page includes documentation for both v1 and v2 Ionic components. Naturally, they show v2 by default. I only just realized this today. 

In hindsight, this explains why I was having a hard time getting some components to work. From now on, I'll be sure to consult the documentation for Ionic v1.

The bottom line is that I got the floating text area working for social network page without breaking anything.

**Comment by bwwagner** [comment]
<a href="https://github.com/bwwagner"><img src="https://avatars4.githubusercontent.com/u/13163930?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [bwwagner](https://github.com/bwwagner)**
_Sunday Mar 05, 2017 at 04:35 GMT_

----

Clayton, do we need to send an AJAX POST to end the broadcast?  If so, what form does that need to take?

If you wouldn't mind giving an idiot's guide on how we are using the POST functionality, I can also try to use it to send the Facebook posts to the server.
